src/components/Todo/TodoStyles.ts

```diff
@@ -0,0 +1,39 @@
+import styled from "styled-components";
+import { Grid, Text } from "@mantine/core";
+
+export const StyledText = styled(Text)``;
+
+export const TodoWrapper = styled(Grid)`
```

Comment:
Na projektech se většinou držíme přístupu, že jeden soubor jedna komponenta. Zároveň pokud neplánuješ stylovat Text, tak ho můžeš použít rovnou nebo ho přeexportovat z komponent, ale bez použití těch styled komponent, aby jsi to měl na jednou místě.